influxql/iterator.go

```diff
@@ -12,13 +11,11 @@ import (
 // ErrUnknownCall is returned when operating on an unknown function call.
 var ErrUnknownCall = errors.New("unknown call")
 
-const (
-	// MinTime is used as the minimum time value when computing an unbounded range.
-	MinTime = int64(0)
+// MinTime is used as the minimum time value when computing an unbounded range.
+const MinTime = int64(0)
 
-	// MaxTime is used as the maximum time value when computing an unbounded range.
-	MaxTime = int64(math.MaxInt64)
-)
+// MaxTime is used as the maximum time value when computing an unbounded range.
+var MaxTime = time.Date(2050, time.January, 1, 0, 0, 0, 0, time.UTC).UnixNano()
```

Comment:
It might be good to just calculate this value and keep it as a constant with a comment saying "This magic number represents this time." That way the style and the godoc output will be more consistent.